.github/workflows/tests.yml

```diff
@@ -117,6 +117,47 @@ jobs:
       - name: Build
         run: yarn build --projects=@strapi/admin,@strapi/helper-plugin
 
+  e2e:
+    timeout-minutes: 60
+    needs: [changes, lint, unit_front, build]
+    name: 'e2e (browser: ${{ matrix.project }})'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        project: ['chromium', 'webkit', 'firefox']
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Monorepo install
+        uses: ./.github/actions/yarn-nm-install
+
+      - name: Install Playwright Browsers
+        # we need 1.37.0 to support Firefox install...
```

Comment:
I lean towards dropping firefox testing and sticking with a stable version to ensure we don't encounter general issues. But I don't have a very strong opinion, if anyone thinks the value of testing an extra browser is more important.

Reply:
IMHO i think it's fine, the benefits of testing firefox outweight using the canary version, i'd expect `1.37` to be released in the coming weeks and we can upgrade then.